packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -258,17 +260,21 @@ export default function FeedItemComponent({
           showImage={!insaneMode}
         />
       );
-    case 'userAcquisition': {
+    case FeedItemType.UserAcquisition:
       return <AcquisitionFormCard key="user-acquisition-card" />;
-    }
-    case 'marketingCta': {
+    case FeedItemType.PublicSquadEligibility:
+      return (
+        <PublicSquadEligibilityCard
+          placeholder={<PlaceholderTag showImage={!insaneMode} />}
+        />
```

Comment:
Just wondering why did you decide to pass the placeholder, wouldn't it be better to do it inside the component?

Reply:
The placeholder depends on the feed, I didn't want to include all that logic again hence we passed it.

Reply:
How does it depend on the feed?
this card is only in squad feed no?

Reply:
There are certain conditions when fetching the placeholder to use, and iirc, we have plans to move the squad feed to use the new layout in the future. I could be wrong though.

<img width="632" alt="Screenshot 2024-05-23 at 3 58 26 PM" src="https://github.com/dailydotdev/apps/assets/13744167/fb165093-bb5c-4780-811f-05599776951a">

Reply:
Ah yeah so for those cases you should actually create 2 different cards with placeholders inside.
It's the better approach, but some system didn't take it into account already.

I see not for all new cards we do use this system.

Reply:
Cool. Yeah, we can fix the problem when we get there. Have update the implementation.

Reply:
squad feed already uses the new layout 😬

Reply:
I will check this before merging.